rfcs/XXXX-gene-sets/text.md

```diff
@@ -0,0 +1,100 @@
+# Gene Sets Data Format
+
+## [Optional] Subtitle of my design 
+
+**Authors:** [Madison Dunitz](mailto:madison.dunitz@chanzuckerberg.com)
+
+**Approvers:** [Friendly Tech Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly PM Person](mailto:some.nice.person@chanzuckerberg.com), [Friendly Tech Person2](mailto:some.nice.person@chanzuckerberg.com), [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com), [Sara Gerber](mailto:sara.gerber@chanzuckerberg.com) 
+
+## tl;dr 
+
+This RFC contains the details about the data format for the gene set feature. It will support the export of differential
+ expression results and allow a user to upload their own gene sets for analysis. 
+ 
+## Glossery 
+- Gene set - List of genes
+- Differential expression - A statistic (typically logfold and p-value) describing the difference in gene expression between two sets of cells. It may refer to stats specific to one gene (differential expression value) or many (differential expression analysis). When referring to many genes it is typically capped (eg top 10 most differentially expressed genes)
+- LogFold - A ratio of the difference in expression levels. Typically calculated by taking the log2 of the gene expression value for each cell and then finding the mean of that value for each set of cells. Because the data is already log2 transformed you can then take the difference between the two means as the logfold value 
+## Problem Statement | Background
+
+Gene sets are the expected output of differential expression analysis in cellxgene. Two (non-overlaping) sets of cells 
+are selected and the difference in the expression level for each gene is computed by comparing the average 
+expression level for each set of cells. When there is a particularly large difference in expression between the two sets of 
+cells, that gene is identified and presented to the user. To ensure consistency and compatibility it is necessary to define a 
+data format for storing references to these genes and any accompanying statistical data. 
+
+## [Optional] Product Requirements
+### Users should be able to export differential expression results
+Currently cellxgene users can choose two sets of cells and compute/display the log fold and adjusted p-value of the top ten differentially expressed genes. Additionally we display a histogram of the gene expression levels of the selected cells for each gene in the right side bar. 
+- [US1] A user should be able to export the list of 10 genes along with their logfold and adjusted p-values in a csv file (see data formats below for more detail) by clicking a button
```

Comment:
Genes or genesets?

Reply:
genes. Currently we calculate the 10 most differentially expressed genes for the set of cells being compared

Reply:
My understand is that genesets would allow more to be exported though. Exporting up to 10 genes per geneset makes sense though.. is that what you meant? A geneset that includes log fold change and p-value would implicitly mean that two groups of cells are being compared for which you want to find the top 10 differentially expressed genes.

cc @signechambers1

Reply:
A user should be able to export a gene set, if the gene set was created by differential expression, the pvalue, log fold change, and category label fields will be populated in addition to the gene name. Once we support differential expression results as a gene set, we will increase the number of results up to 100.

So I think the use case is that a user should be able to export a gene set (not limited to 10 genes).